pkg/k8sutil/sort_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sutil
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
+type testVersions struct {
```

Comment:
Should we test `crdVersions` directly so that in addition to testing the LessVersions function, we're also testing the struct's `Len()`, `Less()`, and `Swap()` functions?

Reply:
Yeah I'm going to move the tests around, let me know what you think when done.